network/sirius/database/genesis.go

```diff
@@ -10,7 +10,9 @@ const Genesis = `
         "eip150Block": 0,
         "eip155Block": 0,
         "eip158Block": 0,
-        "ByzantiumBlock": 0
+        "ByzantiumBlock": 0,
+        "ConstantinopleBlock": 558000,
+        "PetersburgBlock": 558001
```

Comment:
IMO it would be better to use same block number on both

Reply:
yea already changed